projects/gfbio-app/src/app/app.component.ts

```diff
@@ -233,6 +239,7 @@ export class AppComponent implements OnInit, AfterViewInit {
      * @return true, if the splash dialog should be skipped, false otherwise
      */
     private handleQueryParameters(): void {
+        this.tryLogin();
```

Comment:
I need an explanation what the baskets even are to think about this properly.

Reply:
Baskets are GFBio items that are submitted to Geo Engine UI via a URL parameter.
The app will load these items and append them as layers.
If the layer list is not empty, it will show a popup that shows several options.

Reply:
I guess this is correct: 989c122b2d2d8284a699dd4d04cd5e277c912dcb ?